clients/lightrider/lightrider-entry.js

```diff
@@ -25,9 +25,9 @@ const LR_PRESETS = {
  * If configOverride is provided, lrDevice and categoryIDs are ignored.
  * @param {Connection} connection
  * @param {string} url
- * @param {LH.Flags} flags Lighthouse flags, including `output`
+ * @param {LH.Flags} flags Lighthouse flags
```

Comment:
make sure you aren't accidentally still passing in `output: 'html'` and doing all that extra work :)

Reply:
Added a comment.. i dont think we need a stronger check.

Reply:
> Added a comment.. i dont think we need a stronger check.

yeah, I meant just double check existing code, definitely don't need a check *in* the code